packages/cli/src/commands/widgetset/index.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2025 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { type CliCommonArgs, YargsCheckError } from "@osdk/cli.common";
+import type { CommandModule } from "yargs";
+import type { WidgetSetRid } from "../../net/WidgetSetRid.js";
+import configLoader from "../../util/configLoader.js";
+import { logConfigFileMiddleware } from "../../yargs/logConfigFileMiddleware.js";
+import type { CommonWidgetSetArgs } from "./CommonWidgetSetArgs.js";
+import deploy from "./deploy/index.js";
+import { logWidgetSetCommandConfigFileOverride } from "./logWidgetSetCommandConfigFileOverride.js";
+import version from "./version/index.js";
+
+const command: CommandModule<CliCommonArgs, CommonWidgetSetArgs> = {
+  command: "widgetset",
+  describe: "Manage your widget set",
+  builder: async (argv) => {
+    const config = await configLoader("widgetSet");
+    const widgetSet = config?.foundryConfig.widgetSet.rid;
+    const foundryUrl = config?.foundryConfig.foundryUrl;
+    return argv.options({
+      widgetSet: {
```

Comment:
Slight rename from `widgetSet.rid` -> `--widgetSet` to be consistent with site commands which map `site.application` -> `--application`. The foundry.config.json field staying as `rid` seems fine as it's obvious from the context what it refers to